src/cli.rs

```diff
@@ -33,6 +33,11 @@ pub struct Cli {
     #[clap(help_heading = "GLOBAL OPTIONS", global = true)]
     pub manifest_path: Option<PathBuf>,
 
+    /// Path to the audits directory
+    #[clap(long, name = "SUPPLY_CHAIN_DIR", parse(from_os_str))]
+    #[clap(help_heading = "GLOBAL OPTIONS", global = true)]
+    pub config: Option<PathBuf>,
+
```

Comment:
I believe that in the past we've used the term "store" (e.g. in the `[package.metadata.vet]` section) or "supply-chain" (e.g. as the directory name) to refer to this directory, rather than "config". Could we consider making this flag be `--store-path` instead?

This could probably also be documented as the "Path to the supply-chain directory" or similar.

Reply:
Done.